src/lib/install-cli.ts

```diff
@@ -0,0 +1,74 @@
+import { dialog } from 'electron';
+import { mkdir, readlink, symlink, unlink } from 'node:fs/promises';
+import path from 'node:path';
+import { __ } from '@wordpress/i18n';
+import { sudoExec } from 'src/lib/sudo-exec';
+import { getMainWindow } from 'src/main-window';
+import { getResourcesPath } from 'src/storage/paths';
+import packageJson from '../../package.json';
+
+const installedCLIPath = '/usr/local/bin/studio';
+
+const binPath = path.join( getResourcesPath(), 'bin' );
+const packagedPath = path.join( binPath, 'studio-cli.sh' );
+const installScriptPath = path.join( binPath, 'install-studio-cli.sh' );
+
+export async function installCLIWithConfirmation() {
```

Comment:
I think `WithConfirmation` is redundant, sicne it's specific part of installation for macOS. So I would replace this with menthioning `macOS` in the name. And it will be easier to read this funciton name inside `src/menu.ts`. 
```suggestion
export async function installMacOsCLI() {
```

Reply:
Covered in [this PR](https://github.com/Automattic/studio/pull/1141), to simplify understanding my proposition in practice

Reply:
I quite like the `WithConfirmation` suffix. This function is simply a wrapper around `installCLI` with confirmation modals, which the name clarifies.

Reply:
I'm onboard with adding the "macOS" part to the function name, though 👍